pkg/backends/otlp/backend.go

```diff
@@ -66,17 +69,45 @@ func NewClientFromViper(v *viper.Viper, logger logrus.FieldLogger, pool *transpo
 		discarded:             cfg.TimerSubtypes,
 		client:                tc.Client,
 		logger:                logger,
-		sem:                   make(chan struct{}, cfg.MaxRequests),
+		requestsBufferSem:     make(chan struct{}, cfg.MaxRequests),
 	}, nil
 }
 
 func (*Backend) Name() string {
 	return BackendName
 }
 
-func (*Backend) SendEvent(ctx context.Context, e *gostatsd.Event) error {
-	// Events are currently ignored and dropped
-	return nil
+func (b *Backend) SendEvent(ctx context.Context, event *gostatsd.Event) error {
+	se, err := data.NewOtlpEvent(
+		event,
+	)
+	if err != nil {
+		return err
+	}
+
+	el := se.TransformToLog()
+
+	req, err := data.NewEventsRequest(ctx, b.endpoint, el)
+	if err != nil {
+		atomic.AddUint64(&b.droppedEvents, 1)
+		return err
+	}
+
+	b.requestsBufferSem <- struct{}{}
+	resp, err := b.client.Do(req)
+	<-b.requestsBufferSem
+	if err != nil {
+		atomic.AddUint64(&b.droppedEvents, 1)
```

Comment:
If this returns here how does statser know to add it in `backend.dropped_events`

Reply:
same thing for the metrics

Reply:
:kek:

Reply:
I'll raise a github issue :hehehe: